V5.15.12 126 wb

taraant opened the pull request:
Создал ветку v5.15.12-126 - с чистым кодом производителя (т.е. перезатерев все наши изменения)

Отнаследовался от нее и сделал изменения, которые были сделаны нами в ветке v5.13.0.1-112, сгруппировав их по коммитам:
- изменение уровней логгирования, 
- изменения для сборки под наши ядра, 
- настройка Makefile для нашей платформы

taraant-wb committed:
Added README.md

taraant-wb committed:
Log output adjustments

taraant-wb committed:
Fixes for building with kernels 6.8 and 5.10

taraant-wb committed:
WiFi configuration change for WirenBoard boards

KraPete commented on Makefile:
```diff
@@ -190,7 +192,8 @@ CONFIG_SECURE_DMA = n
 CONFIG_SECURE_DMA_MEM_SIZE = 3686400
 ###################### Platform Related #######################
-CONFIG_PLATFORM_I386_PC = y
+CONFIG_PLATFORM_WB7 = y
```
Тут уже можно назвать `CONFIG_PLATFORM_WB`, теперь же не только WB7

taraant replied:
ok

KraPete commented on Makefile:
```diff
@@ -1466,6 +1469,19 @@ EXTRA_CFLAGS += -DCONFIG_RTW_80211R
+EXTRA_CFLAGS += -DCONFIG_IOCTL_CFG80211 -DRTW_USE_CFG80211_STA_EVENT
+EXTRA_CFLAGS += -DCONFIG_CONCURRENT_MODE
+EXTRA_CFLAGS += -DCONFIG_RADIO_WORK
+ARCH := arm
```
Для WB8 тут не надо что-то другое? У нас же aarch64, а на WB7 - armhf.

taraant replied:
да, ты прав. 
Видимо переменные окружения приоритетнее чем это, или при компиляции ядра сюда вообще не смотрится в этом плане. Попробую вообще убрать.

KraPete replied:
Может, тогда сделать два конфига для WB7 и WB8. В них корректно прописать компилятор и ARCH. А в конфиге конкретного ядра выбирать то, что надо?

taraant replied:
Решили что раз мы не компилим модуль отдельно и используется компилятор ядра, то вообще убрать упоминание компилятора, чтобы не вводить в заблуждение.

taraant-wb committed:
Redesigned the WB7 platform in favor of the universal WB.
Changed the compiler version for separate compilation

taraant-wb committed:
Removed the mention of the compiler.
Since we don’t have scenarios for compiling the module outside the kernel, I removed the mention of the compiler to avoid confusion

KraPete reviewed (approved):
Хочу в результате, чтоб последние 3 коммита шли как один
